tests/test_simplex_collection.cpp

```diff
@@ -1357,3 +1357,144 @@ TEST_CASE("are_simplex_collections_equal", "[simplex_collection]")
     sc2.sort_and_clean();
     REQUIRE(SimplexCollection::are_simplex_collections_equal(sc1, sc2) == true);
 }
+
+
+TEST_CASE("simplex_link_condtion_tetmesh", "[simplex_collection]")
+{
+    SECTION("one tet")
+    {
+        tests_3d::DEBUG_TetMesh m;
+        m = tests_3d::single_tet();
+        std::array<Tuple, 6> e;
+        // cannot collapse
+        e[0] = m.edge_tuple_from_vids(0, 1);
+        e[1] = m.edge_tuple_from_vids(0, 2);
+        e[2] = m.edge_tuple_from_vids(0, 3);
+        e[3] = m.edge_tuple_from_vids(1, 2);
+        e[4] = m.edge_tuple_from_vids(1, 3);
+        e[5] = m.edge_tuple_from_vids(2, 3);
+
+        for (int i = 0; i < 6; ++i) {
+            REQUIRE(link_condition(m, e[i]) == false);
```

Comment:
`CHECK` instead of `REQUIRE` also in the rest of the test.

Instead of writing `... == false` you can either write `!...` or `CHECK_FALSE(...)`